### PR TITLE
Add Gitter to navbar and pull external links right

### PR DESCRIPTION
**Pull request by MattMS**
This still only updates the home and docs pages.
I need to confirm the appropriate process for updating the repl page.

### CODE REVIEW

File: docs/index.html.handlebars [hunk 1]
```diff
@@ -9,19 +9,22 @@
...
+                </a>
+            </div>
+            <ul class="nav navbar-nav navbar-left">
+                <li><a href="../">Home</a></li>
                 <li class="active"><a href="#">Documentation</a></li>
                 <li><a href="/repl?v={{version}}">Try Ramda</a></li>
+            </ul>
+            <ul class="nav navbar-nav navbar-right">
                 <li><a href="https://github.com/ramda/ramda">GitHub</a></li>
+                <li><a href="https://gitter.im/ramda/ramda">Gitter</a></li>
```
Comment (buzzdecafe) [inline docs/index.html.handlebars]: I'd prefer "Discuss" to "Gitter", but curious if others have preferences?

**Comment by buzzdecafe** [comment]
> I need to confirm the appropriate process for updating the repl page.

good question -- i think it is hard-coded in repl/index.html; so maybe another enhancement would be to share a header/footer/template. But that is not urgent, nor even required for this PR

File: docs/index.html.handlebars (hunk 1 above)
Comment (MattMS) [inline docs/index.html.handlebars]: That would make it easier to spot beside "GitHub".

Would the use of "Gitter" elsewhere make it seem like "Discuss" goes somewhere else?

I'll also offer "Chat" as an alternative, to match the "Open chat" Gitter Sidecar link.
That said, I have no major preference between any of them.

**Comment by MattMS** [comment]
I did start thinking about putting the header in a template but then noticed there is the slight complexity of highlighting the current page.
It was around 2am so I decided to take the lazy approach.

I'm curious about the Handlebars includes now, so I'll likely make another PR once I figure out how to do them, unless someone beats me to it.

File: docs/index.html.handlebars (hunk 1 above)
Comment (buzzdecafe) [inline docs/index.html.handlebars]: Yeah, i was looking for a way to change "Open Chat" to "Discuss" as well :smile:; but i didn't see an obvious way from glancing at the sidecar API

Comment (MattMS) [inline docs/index.html.handlebars]: Updated to "Discuss".

**Comment by buzzdecafe** [comment]
No one else has weighed in on the text, so I am going to take that as tacit agreement :smile:

:cow2:

**Pull request merged**